src/ansys/geometry/core/designer/body.py

```diff
@@ -867,6 +904,38 @@ def translate(  # noqa: D102
             )
         )
 
+    @protect_grpc
+    @check_input_types
+    @reset_tessellation_cache
+    def set_name(  # noqa: D102
+        self, name: str
+    ) -> None:
+        self._grpc_client.log.debug(f"Translating body {self.id}.")
+        self.name = name
+        self._bodies_stub.SetName(
+            SetNameRequest(
+                body_id=self.id,
+                name=name,
+            )
+    )
+
+    @protect_grpc
+    @check_input_types
+    @reset_tessellation_cache
+    def set_fill_style(  # noqa: D102
+        self, fill_style: FillStyle
+    ) -> None:
+
+        self._grpc_client.log.debug(f"Set body fill style{self.id}.")
+        self._fill_style = fill_style
+        self._bodies_stub.SetFillStyle(
+            SetFillStyleRequest(
+                body_id=self.id,
+                fill_style=fill_style.value,
+            )
+        )
+
+
```

Comment:
The "name" setter property and the "fill_style" setter property should delegate to "set_*". Please implement it.

Reply:
can you elaborate this more?

Reply:
I thought it might be easier to just show you what I meant. See https://github.com/ansys/pyansys-geometry/pull/1299/commits/313273d867b4920c488ecb23a873c0c5a4da371d